kview/src/main/java/dev/ikm/komet/kview/mvvm/view/export/ExportController.java

```diff
@@ -242,6 +244,7 @@ private PopOver createPopover(UUID exportTopic, final int rangeType, Consumer<Lo
     public void handleCurrentDateTimeExport() {
         // Responsible for showing/hiding custom date range controls
         timePeriodComboBox.valueProperty().addListener((obs, oldVal, newVal) -> {
+            //FIXME
```

Comment:
Can you make this into a ToDo comment and add more details on what needs to be fixed ?

Reply:
this comment is old and will remove it.